extensions/civetweb/processors/ListenHTTP.cpp

```diff
@@ -390,7 +394,25 @@ bool ListenHTTP::Handler::handleGet(CivetServer *server, struct mg_connection *c
   return true;
 }
 
-void ListenHTTP::Handler::write_body(mg_connection *conn, const mg_request_info *req_info) {
+bool ListenHTTP::Handler::handleHead(CivetServer *server, struct mg_connection *conn) {
```

Comment:
Our processor documentation says we serve HEAD requests, and NiFi does it at well, so we should too.